beacon_node/rest_api/src/validator.rs

```diff
@@ -28,8 +28,8 @@ pub struct ValidatorDuty {
     pub attestation_committee_index: Option<CommitteeIndex>,
     /// The position of the validator in the committee.
     pub attestation_committee_position: Option<usize>,
-    /// The slots in which a validator must propose a block (can be empty).
-    pub block_proposal_slots: Vec<Slot>,
+    /// The slot in which a validator must propose a block, or `null` if block production is not required.
+    pub block_proposal_slot: Option<Slot>,
```

Comment:
This seems to be undoing #662, and the squish commit seems to be the issue. `git checkout -p origin/master -- <files>` might be the quickest way back to normality. Or you could replay #662 on top to redo its changes?